.circleci/build_steps.sh

```diff
@@ -35,7 +37,12 @@ source run_conda_forge_build_setup
 /usr/bin/sudo -n yum install -y devtoolset-2-gcc-gfortran
```

Comment:
This line should be removed by re-rendering now that `yum_requirements.txt` is removed, but it is not. Thoughts @conda-forge/core?

Reply:
The bot's copy of `conda-forge-pinning` was out-of-date, which caused re-rendering to crash. Instead of letting us know it had an error, it merely stated that re-rendering was a no change. This seems like a bug in the bot. Have redeployed the re-rendering bot to fix this.

Reply:
Raised as issue ( https://github.com/conda-forge/conda-forge-webservices/issues/230 ).